### ISSUE TITLE
Lower badge strings that reflect numbers are converted to numbers

### ISSUE THREAD

**Issue by mbehr1** [mbehr1/fishbone#9]
E.g. a string '1101' returned from a javascript function should be shown in lower Badge as "1101" but is shown as "99+".
If a javascript function returns a string it should be used as if.

Version: 1.8.1

**Comment by gerstej9** [comment]
Hi mbehr1,

I was trying to work on this issue and ran into some issues with the initial build. I tried following the build.md and vsc-extension quick start but with no success. I did have some luck when I strictly used a yarn install after cloning the repo. I wanted to let you know in the hope that it helps you with your documentation. I also believe I have identified where this issue might be coming from. Based on what I can tell the issue is coming from lines 186-187 in the fbaCheckbox.js file within ./src/webview. ` if ('result' in res) {
                        setBadge2Counter(res.result);
                        setBadge2Status(2);`
While I realize this is not a solved issue, I do hope this information helps you moving forward.

**Comment by mbehr1** [comment]
@gerstej9 thx! I'll check the install / build.md docs again!

**Comment by mbehr1** [comment]
I think the problem is inside the material-ui badge component. The code in https://github.com/mui-org/material-ui/blob/d72862cf60b2ea314b5d9e3ed163a0f605e2eef2/packages/material-ui/src/Badge/Badge.js#L186 does something like:
```
let max=99;
let badgeContent = "1101";
let displayValue = badgeContent > max ? `${max}+` : badgeContent;
console.log(`${displayValue}`, typeof badgeContent); // gives "99+", "string"
```

**Comment by mbehr1** [comment]
to avoid these I think it's easiest to set max to "infinite"/NaN... and do this ">99" check "properly" (i.e. only for typeof === 'number') outside the object.

**Comment by gerstej9** [comment]
Hi mbehr1, 
That does look more like it than what I came across. I was having a hard time since I couldn't get it up and running and am new to open source contributions but wanted to take a crack at it. Thank you for sharing the solution you found.

**Comment by mbehr1** [comment]
@gerstej9  no worries! thx anyhow!

**Comment by mbehr1** [comment]
fixed with v1.8.2
